spec/abci/abci.md

```diff
@@ -5,27 +5,32 @@ title: Method and Types
 
 # Methods and Types
 
-## Overview
-
-The ABCI message types are defined in a [protobuf
-file](https://github.com/tendermint/tendermint/blob/master/proto/tendermint/abci/types.proto).
-
-ABCI methods are split across four separate ABCI _connections_:
-
-- Consensus connection: `InitChain`, `BeginBlock`, `DeliverTx`, `EndBlock`, `Commit`
-- Mempool connection: `CheckTx`
-- Info connection: `Info`, `Query`
-- Snapshot connection: `ListSnapshots`, `LoadSnapshotChunk`, `OfferSnapshot`, `ApplySnapshotChunk`
-
-The consensus connection is driven by a consensus protocol and is responsible
-for block execution.
-
-The mempool connection is for validating new transactions, before they're
-shared or included in a block.
-
-The info connection is for initialization and for queries from the user.
-
-The snapshot connection is for serving and restoring [state sync snapshots](apps.md#state-sync).
+## Connections
+
+ABCI applications can run either within the same _process_ as the Tendermint 
+state-machine replication engine _or_ as a separate process from the state-machine
```

Comment:
Minor nit: It feels like we're emphasizing different things here.

```suggestion
ABCI applications can run either within the _same_ process as the Tendermint 
state-machine replication engine, or as a _separate_ process from the state-machine
```

Reply:
reread this just now and noticed the same thing.